packages/@okta/vuepress-site/docs/guides/oie-upgrade-plan-embedded-upgrades/main/index.md

```diff
@@ -96,15 +101,15 @@ When planning how to roll the upgrade out to your users, take the following sugg
     * Have logic in your code to send some users into the Classic Engine workflow and some to Identity Engine.
     * Load balance network traffic to different application instances, containers, or web servers to send some to a Classic Engine version of your app or to an Identity Engine version.
     * Increase this over time, as you see fit.
-    * Remove code or shutdown alternate versions when everything is working.
+    * Remove code or shutdown alternate versions when everything is functional.
 
-## Troubleshooting
+## Troubleshoot
 
-This section details common issues and errors you may encounter during your upgrade, and likely causes.
+This section details common issues and errors that you may encounter during your upgrade, and likely causes.
 
 | Issue / error | Cause |
 | ------ | -------------------------- |
-| Error message: "The requested feature is not available in this environment" | This is reported when your [Okta org](/docs/concepts/okta-organizations/) hasn't been upgraded to Identity Engine and you try to turn on [interaction code](/docs/concepts/interaction-code/) in the embedded SDK/Sign-In Widget. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version. |
-| &nbsp; | The above error is also reported when you *have* upgraded your org to Identity Engine and interaction code is turned on in the embedded SDK, but it isn't turned on in your app or authorization server. |
+| Error message: "The requested feature isn't available in this environment" | This is reported when your [Okta org](/docs/concepts/okta-organizations/) hasn't been upgraded to Identity Engine and you try to turn on [interaction code](/docs/concepts/interaction-code/) in the embedded SDK/Sign-In Widget. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version. |
```

Comment:
Maybe something like:
```suggestion
| Error message: "The requested feature isn't available in this environment" | This is reported when you attempt to enable [interaction code](/docs/concepts/interaction-code/) in the embedded SDK/Sign-In Widget on a non-OIE [Okta org](/docs/concepts/okta-organizations/). See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version. |
```

Reply:
I'm going to leave this one as is - we aren't supposed to use OIE in outward facing docs. It's an internal term.